pkg/auth/jwt/auth.go

```diff
@@ -131,8 +131,7 @@ func (JA *JWTauth) Refresh(token []byte) ([]byte, error) {
 		BaseURL(JA.authURL).
 		Encoder(json.Marshal).
 		Decoder(request.JSONDecoderGenerator).
-		Header("Authorization", fmt.Sprintf("Bearer %v", string(token))).
-		Build()
```

Comment:
Why remove Build()?

Reply:
`Build()` was removed because the way the request client is created has changed, it's no longer a builder that at with the function Builder returns the pointer to the struct that was changed.

I changed the implementation so that the struct at each of these functions create a copy of themselves, that makes the build useless since all the other functions already return the client without being a pointer. For the cases in which a pointer is needed there is a new method called Pointer that returns the memory address of the struct created.